src/test/java/nl/tudelft/contextproject/script/ScriptTest.java

```diff
@@ -81,7 +85,7 @@ public void testScript() {
         assertEquals(script1.getShots().get(0).getCamera(), cam0);
         assertEquals(script1.getShots().get(0).getNumber(), 1);
         assertEquals(script1.getShots().get(0).getPreset(), pres);
-        assertFalse(script1.getTimeline(cam0.getNumber()) == script1.getTimeline(cam1.getNumber()));
+        assertNotEquals(script1.getTimeline(cam0.getNumber()), script1.getTimeline(cam1.getNumber()));
         assertEquals(script1.getTimeline(cam0.getNumber()).getCamera(), cam0);
```

Comment:
Well this method has a lot of parameters which all have to be checked and as such, cannot be separated.